predicate_test.go

```diff
@@ -0,0 +1,45 @@
+package opslevel_test
+
+import (
+	"encoding/json"
+	"testing"
+
+	ol "github.com/opslevel/opslevel-go/v2024"
+	"github.com/rocktavious/autopilot/v2023"
+)
+
+func TestJsonMarshalPredicateUpdateInputNull(t *testing.T) {
```

Comment:
Ahh this isn't want I meant but is a good test to have too.  What I wanted to see was the actual marshaling of check input as done by the graphql framework.  Like a test on the Check type that uses predicates.